src/routes/event/add.tsx

```diff
@@ -18,39 +19,88 @@ export default function EventAdd() {
     });
   }, []);
   const [location, setLocation] = React.useState<string | null>(null);
+  const [category, setCategory] = React.useState<string | null>(null);
```

Comment:
you can just do this `useState("")` it will default to string type
and empty string === false;